src/Core/Components/NumberField/FluentNumberField.razor.cs

```diff
@@ -173,5 +184,34 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
             }
 
         }
+        else
```

Comment:
I don't like calling a JS method every time a render is raised :-(

Why not use a JS Observable to apply the `value` attribute to the `value` JS property?
Like this: https://github.com/microsoft/fluentui-blazor/blob/dev-v5/src/Core/Components/TextInput/FluentTextInput.razor.ts

Reply:
I'll take a look at that approach 🙏

Reply:
@dvoituron 

This was a great suggestion 😄.  It's definitely better than my original approach.